app/views/users/sessions.py

```diff
@@ -141,6 +144,65 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        for speaker in session.speakers:
+            data += (speaker.name + ',') if speaker.name else ','
+            data += (session.title + ',') if session.title else ','
+            data += (speaker.email + ',') if speaker.email else ','
+            data += (speaker.sessions + ',') if speaker.sessions else ','
+            data += (speaker.mobile + ',') if speaker.mobile else ','
+            data += (speaker.organisation + ',') if speaker.organisation else ','
```

Comment:
Going to use **csv** class and write it using **writerow** function from the module  
Also it needs to be done in [this also ](https://github.com/fossasia/open-event-orga-server/pull/3390) for the issue of [**Implement "Download Attendees List" and "Download Orders List"**](https://github.com/fossasia/open-event-orga-server/issues/3246) 

Anything different you would suggest ?